spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -619,6 +619,19 @@ dataflow:>task launch my-composed-task --arguments "--increment-instance-enabled
 Launched task 'my-composed-task'
 ----
 
+===== Launching a Composed Task using Custom Composed Task Runner
+
+In some cases a user will need to launch a composed task using a Composed Task Runner other than the one specified with `spring.cloud.dataflow.task.composedTaskRunnerName` property.
```

Comment:
To make it more descriptive, what if we show the registration of a custom CTR and then highlight its use of it at the time of launch?

I had dropped an example in the issue description: https://github.com/spring-cloud/spring-cloud-dataflow/issues/3377

Reply:
Sorry, one other suggestion. Can we rephrase this to make the customized version of CTR stand out? Something like:

> In some cases a user will need to launch a composed task using a custom version of a Composed Task Runner other than default application that is shipped out-of-the-box.